insight/tasks/weekly_newsletter_batch.py

```
@@ -0,0 +1,467 @@
+                            e_date=self.weekly_info['e_date'], 
+                            is_expired_token_user=is_expired,
+                            weekly_trend_html=weekly_trend_html, 
+                            user_weekly_trend_html=user_weekly_trend_html, 
+                        ))
+                    )
+                    text_body = html2text.HTML2Text().handle(html_body)
+                    
+                    newsletter = Newsletter(
+                        user_id=user['id'],
+                        email_message=EmailMessage(
+                        to=[user['email']],
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)